cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -25,6 +26,13 @@
     "map": "map",
     "name": "name",
     "regex": "regularExpression",
+    "section": "section",
+    "one section": "sectionLevelOne",
+    "two section": "sectionLevelTwo",
+    "three section": "sectionLevelThree",
+    "four section": "sectionLevelFour",
+    "five section": "sectionLevelFive",
+    "six section": "sectionLevelSix",
```

Comment:
Any better ideas for spoken forms here?  The obvious "section one" results in ambiguous grammar if there's a hat over a `1`.  Could do "section level one"?  "level one section"?